index.js

```diff
@@ -9,14 +9,14 @@ const bundler = require('./bundler')
 
 const windows = common.windows
 
-const inputDefaults = {
-  'ruby-version': 'default',
-  'rubygems': 'default',
-  'bundler': 'Gemfile.lock',
-  'bundler-cache': 'false',
-  'working-directory': '.',
-  'cache-version': bundler.DEFAULT_CACHE_VERSION,
-}
```

Comment:
We cannot remove this, this is useful for actions like https://github.com/ruby/setup-ruby-pkgs.